components/clp-py-utils/clp_py_utils/create-results-cache-indexes.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import logging
+import pathlib
+import sys
+
+import pymongo
+
+# Setup logging
+# Create logger
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+# Setup console logging
+logging_console_handler = logging.StreamHandler()
+logging_formatter = logging.Formatter("%(asctime)s [%(levelname)s] %(message)s")
+logging_console_handler.setFormatter(logging_formatter)
+logger.addHandler(logging_console_handler)
+
+from clp_py_utils.clp_config import ResultsCache
+from clp_py_utils.core import read_yaml_config_file
+
+
+def main(argv):
+    args_parser = argparse.ArgumentParser(description="Creates results cache indexes for CLP.")
+    args_parser.add_argument("--config", required=True, help="Results cache config file.")
+    parsed_args = args_parser.parse_args(argv[1:])
+
+    config_file_path = pathlib.Path(parsed_args.config)
+
+    try:
+        results_cache_config = ResultsCache.parse_obj(read_yaml_config_file(config_file_path))
+        if results_cache_config is None:
+            raise ValueError(f"results cache configuration file '{parsed_args.config}' is empty.")
+
+        results_cache_uri = results_cache_config.get_uri()
+        ir_collection_name = results_cache_config.ir_collection_name
+
+        with pymongo.MongoClient(results_cache_uri) as results_cache_client:
+            ir_collection = results_cache_client.get_default_database()[ir_collection_name]
+            ir_collection.create_index(["file_split_id"])
```

Comment:
I leave them as constant to follow database creation scripts. If needed, we can create a constant.py for both database and results_cache script